### PR TITLE
chore: Update the text inconsistencies

### PR DESCRIPTION
**Pull request by tmastrom**
Update text inconsistencies 

[card](https://github.com/bcgov/cas-cif/issues/1029)

### CODE REVIEW

**Review by BCerki — COMMENTED** [review]
I've got a couple questions about the AC too and I'll put them in the ticket

File: app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx [hunk 1]
```diff
@@ -273,7 +275,7 @@ describe("the emission intensity report form component", () => {
     ).toHaveTextContent("0.00%");
     expect(
       screen.getByLabelText(
-        "Payment percentage of performance milestone amount"
+        /Payment percentage of performance milestone amount/i
```
[the remark this answers is not in the document]
Comment (tmastrom) [reply app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx:278]: This is something I was wondering during this refactor. I am in favour of making testing case agnostic so these small changes don't also require updates to unit tests.

File: app/cypress/integration/cif/project-revision/index.spec.js [hunk 2]
```diff
@@ -41,21 +41,25 @@ describe("the new project page", () => {
     cy.findByRole("button", { name: /add a secondary contact/i }).click();
     cy.get('[placeholder="Select a Contact"]').should("have.length", 4);
 
-    cy.happoAndAxe("Project contacts Form", "empty", "main");
-    cy.findByText(/Submit contacts/i).click();
+    cy.happoAndAxe("Project Contacts Form", "empty", "main");
+    // Error: unable to find "Submit contacts"
```
[the remark this answers is not in the document]
Comment (tmastrom) [reply app/cypress/integration/cif/project-revision/index.spec.js:45]: Leftover

File: app/cypress/support/commands.js [hunk 3]
```diff
@@ -265,11 +265,8 @@ Cypress.Commands.add(
     // Extra assertion to wait for the new milestone report to be added
     cy.contains("Changes saved").should("be.visible");
 
-    cy.setDateInPicker(
-      "Measurement period start date",
-      measurementPeriodStartDate
-    );
-    cy.setDateInPicker("Measurement period end date", measurementPeriodEndDate);
+    cy.setDateInPicker("TEIMP Start Date", measurementPeriodStartDate);
```
[the remark this answers is not in the document]
Comment (tmastrom) [reply app/cypress/support/commands.js:268]: Refactoring this probably isn't worth the effort. This name is carried through to the graphql schema

[the remark this answers is not in the document]
Comment (BCerki) [reply app/cypress/support/commands.js:270]: I had meant renaming only in `commands.js`, but yes, upon reflection, if there's no connection in this file to what it's actually called, that's probably more confusing

File: app/components/TaskList/index.tsx [hunk 4]
```diff
@@ -228,12 +228,12 @@ const TaskList: React.FC<Props> = ({
...
                 .sectionNumber + 1
             )}
-            listItemName="Submit changes"
+            listItemName="Submit Changes"
           >
             <TaskListItem
               stepName="summary"
               linkUrl={getProjectRevisionPageRoute(id)}
-              formTitle="Review and submit information"
+              formTitle="Review and Submit information"
```
Comment (mikevespi) [inline app/components/TaskList/index.tsx:236]: Information, missed the capital i

**Review by BCerki — COMMENTED** [review]
Still some lowercase/reference to funding agreement on the summary page:
image

**Review by mikevespi — COMMENTED** [review]
Just the one missing capital letter

File: app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (BCerki) [reply app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx:278]: I've added this one to Promyze for discussion next time we have a session. If you want to get the plugin too it's here: https://chrome.google.com/webstore/detail/promyze-for-code-review/oboemiockcgekjobilpopcfpepdojbog/related?hl=en

**Comment by tmastrom** [comment]
> 

Should the "not updated" part be capitalized too?

**Review by BCerki — APPROVED** [review]

**Review by BCerki — APPROVED** [review]

**Pull request merged**